packages/heroku.rb

```diff
@@ -3,15 +3,27 @@
 class Heroku < Package
   description 'The Heroku Command Line Interface (CLI), formerly known as the Heroku Toolbelt, is a tool for creating and managing Heroku apps from the command line / shell of various operating systems.'
   homepage 'https://devcenter.heroku.com/articles/heroku-cli'
-  version '4.28.3'
-  binary_url ({
-    armv7l: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxOHFIMzQyNHNFUWc",
-    i686: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxd3NULTRkMWlHQTA",
-    x86_64: "https://drive.google.com/uc?export=download&id=0ByCixsDmZPzxLURkMktpREpDZk0"
-  })
-  binary_sha256 ({
-    armv7l: "b7fab53d3c8cfd42d41c1b07db069c9fd5f7261fcea48fd99114981cf1a293d7",
-    i686: "1e73fbff3a10c0c95dd1aa1cf68952a95a330634fa410765962df0396315a42c",
-    x86_64: "9259cd8b5d7aaeb9172142b5956ad767a3d4f1bcf126e51a7f4a8c055e53c068"
-  })
+  version '6.13.6'
+  source_url 'https://github.com/heroku/cli/archive/v6.13.6.tar.gz'
+  source_sha256 '395c7e22d16b1fce2d5ed4ac1e022dbd3c9aa8fdbc172d9dadf86ba9c1ad6978'
+
+  depends_on 'nvm'
+  depends_on 'yarn'
+
+  def self.build
+    system '\. ~/.bashrc && nvm install stable && yarn install'
+  end
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/share/heroku"
+    system "cp -r . #{CREW_DEST_DIR}#{CREW_PREFIX}/share/heroku"
+    FileUtils.cd("#{CREW_DEST_DIR}#{CREW_PREFIX}/bin") do
+      system "ln -s /usr/local/share/heroku/bin/run heroku"
+    end
+    puts ""
+    puts "To finish the installation, execute the following:".lightblue
+    puts "source ~/.bashrc".lightblue
```

Comment:
Installing `nvm` is not necessary if node v8.1.4 or above is isntealled already.  In this PR, you assume many ppl don't install new node yet.  It may correct atm.  But, how about 3 months later?

Regarding to make `heroku` depend on `nvm`, there are many other version management tools for node like nodebrew, nbenv, anyenv, ...  I'm still not sure what is the best way to solve this situation.  But, choosing only one version management tool is not the best way I think...